src/Core/src/Platform/iOS/ApplicationExtensions.cs

```diff
@@ -62,10 +62,10 @@ public static void CreateNativeWindow(this UIWindowSceneDelegate sceneDelegate,
 			}
 
 			var window = CreateNativeWindow(application, scene as UIWindowScene, dicts.ToArray());
-			if (window is not null)
+			if (window is not null && sceneDelegate is UIWindowSceneDelegate windowSceneDelegate)
```

Comment:
If the MAUI Delegate is IUIWindowSceneDelegate and there's a cast check for UIWindowSceneDelegate, would this cast work? Shouldn't it be `IUIWindowSceneDelegate`?

Reply:
Yeah same as above comment, switched to use Get/SetWindow calls... Thanks for your eyes on this, wanted to be sure this didn't impact hot reload :)

Reply:
Yep, I think you're good then. I don't think this should impact Hot Reload, as far as I can see.